styleguide/derek/view-templates/events/events.js

```diff
@@ -0,0 +1,3 @@
+jQuery(document).ready(() => {
+  jQuery('.rsFilter-bar').rsFilterBar();
+});
```

Comment:
For consistency, do you wanna swap this internal jQuery reference out like this? 
```
jQuery(document).ready(($) => {
  $('.rsFilter-bar').rsFilterBar();
});
```